### ISSUE TITLE
choosing reactor howto should explain how to install and then get the reactor

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3253]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3253|
|Type|enhancement|
|Created|2008-05-29 12:23:41Z|
|Branch|https://github.com/twisted/twisted/tree/install-reactor-3253|

It doesn't really explain that the way you get it is by doing 'from twisted.internet import reactor' after calling install(). I'm tempted to suggest that every single example in the document should just include the reactor import at the end.

<details><summary>Searchable metadata</summary>

```
trac-id__3253 3253
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__ 
branch__branches_install_reactor_3253 branches/install-reactor-3253
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1212063821000000 1212063821000000
changetime__1252893475000000 1252893475000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

The  ticket description is a little confusing, right now the samples look like:

```python
from twisted.internet import selectreactor
selectreactor.install()
```

but what do you suggest?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [27265]) Branching to 'install-reactor-3253'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

So I assume the samples should look like:

```python
from twisted.internet import selectreactor
selectreactor.install()

from twisted.internet import reactor
```

Also fixed some typo's and a outdated link. Putting it up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Please jam a trailing `/` onto the new pygtk uri (`http://foo` isn't a legal uri, `http://foo/` is).  Aside from that, great, though I can't help wondering if this duplication is the most effective way of conveying the idea.  Oh well.  Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

(In [27326]) Address review comments , refs [#3253](https://github.com/twisted/twisted/issues/3253)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27327]) Merge install-reactor-3253: Include the reactor imports after calling install() and cleanup document.

Author: thijs
Reviewer: exarkun
Fixes: [#3253](https://github.com/twisted/twisted/issues/3253)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
